app/src/main/java/org/oppia/app/player/state/itemviewmodel/DragDropInteractionContentViewModel.kt

```diff
@@ -0,0 +1,9 @@
+package org.oppia.app.player.state.itemviewmodel
+
+import org.oppia.app.model.StringList
+import org.oppia.app.viewmodel.ObservableViewModel
+
+/** [ObservableViewModel] for DragDropSortInput values. */
+class DragDropInteractionContentViewModel(
```

Comment:
Is this class actually used/needed? It seems like we're binding HTML content directly.

Reply:
This is being used inside the custom view